examples/blogs__use-react-devtools/package.json

```diff
@@ -5,11 +5,12 @@
   "scripts": {
     "cypress:open": "../../node_modules/.bin/cypress open",
     "cypress:run": "../../node_modules/.bin/cypress run",
+    "cypress:run:chrome": "../../node_modules/.bin/cypress run --browser chrome",
     "cypress:run:record": "../../node_modules/.bin/cypress run --record",
     "dev": "../../node_modules/.bin/start-test 3000 cypress:open",
     "start": "../../node_modules/.bin/react-scripts start",
     "test:ci": "../../node_modules/.bin/start-test 3000 cypress:run",
-    "test:ci:chrome": "../../node_modules/.bin/start-test 3000 cypress:run",
+    "test:ci:chrome": "../../node_modules/.bin/start-test 3000 cypress:run:chrome",
```

Comment:
This change is kinda unrelated. This test was run as part of our chrome suite, but I noticed that it was being executed against electron there too. So now we run against chrome when we expect to.